src/Commands/AppConfig/KeyValue/KeyValueLockCommand.cs

```diff
@@ -35,6 +35,8 @@ public override async Task<CommandResponse> ExecuteAsync(CommandContext context,
                 return context.Response;
             }
 
+            AddSubscriptionInformation(context.Activity, options);
```

Comment:
I would like to see us have a base.ExecuteAsync() call which BindOptions, Validate and set up stuff like this.